cellprofiler/modules/measureimageskeleton.py

```diff
@@ -0,0 +1,156 @@
+import cellprofiler.module
+import cellprofiler.setting
+import numpy
+import scipy.ndimage
+import skimage.measure
+import skimage.segmentation
+import skimage.util
+
+
+def _neighbors(image):
+    padding = skimage.util.pad(image, 1, "constant")
+
+    mask = padding > 0
+
+    padding = padding.astype(numpy.float)
```

Comment:
I use `astype(numpy.float)` to ensure `scipy.ndimage.uniform_filter` returns a `numpy.float`.